src/data/api/fetch.ts

```diff
@@ -133,6 +134,15 @@ export const useRegistrering = (fnr?: string) => {
     return { data, isLoading, error };
 };
 
+export const useEndringIRegistrering = (fnr?: string) => {
+    const { data, error, isLoading } = useSWR<EndringIRegistreringsdata, ErrorMessage>(
+        fnr ? `/veilarbperson/api/person/registrering/endringer?fnr=${fnr}` : null,
```

Comment:
Since we want to remove fnr from GET requests soon, is is possible to change it to be POST or something else..

Reply:
Done: see https://github.com/navikt/veilarbperson/pull/277